hikari/events/guild_events.py

```diff
@@ -632,17 +635,6 @@ def guild_id(self) -> snowflakes.Snowflake:
         """
         return self.presence.guild_id
 
-    # TODO: make this nicer, as it is inconsistent with stuff elsewhere I guess.
-    def get_cached_user(self) -> typing.Optional[users.User]:
```

Comment:
If we're removing this shouldn't `fetch_user` also be removed? (plus wouldn't a rename to `get_user` also work?

Reply:
Yea, could rename it